wikichange/src/manifest.json

```diff
@@ -3,17 +3,18 @@
     "short_name": "__MSG_appShortName__",
     "description": "__MSG_appDescription__",
     "homepage_url": "http://chrome.google.com/webstore",
-    "version": "0.0.1",
-    "version_name": "0.0.1",
+    "version": "1.0.0",
+    "version_name": "1.0.0",
     "manifest_version": 3,
     "default_locale": "en",
     "minimum_chrome_version": "88",
     "icons": {
-        "16": "assets/icons/wikiIcon16.png"
+        "16": "assets/icons/icon16.png",
+        "48": "assets/icons/icon16.png"
     },
     "action": {
         "default_icon": {
-            "16": "assets/icons/wikiIcon16.png"
+            "16": "assets/icons/icon_16.png"
```

Comment:
There are three places where you are using the icon.png. Need to match them all.